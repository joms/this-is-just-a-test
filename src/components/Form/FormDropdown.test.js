import React from 'react';
import { shallow } from 'enzyme';
import { FormDropdown } from '.';

describe('<FormDropdown>', () => {
    it('matches snapshot', () => {
        const onChange = jest.fn();
        const wrapper = shallow(
            <FormDropdown
                label="Hello, Dropdown!"
                options={[{ value: 3.14, text: 'PI' }, { value: 6.28, text: 'Tau' }]}
                onChange={onChange}
            />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
