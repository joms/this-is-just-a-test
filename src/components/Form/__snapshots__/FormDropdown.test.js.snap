// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormDropdown> matches snapshot 1`] = `
<Field
  horizontal={false}
>
  <Dropdown
    error=""
    hint=""
    label="Hello, Dropdown!"
    onChange={[MockFunction]}
    options={
      Array [
        Object {
          "text": "PI",
          "value": 3.14,
        },
        Object {
          "text": "Tau",
          "value": 6.28,
        },
      ]
    }
    placeholder=""
    value=""
  />
</Field>
`;
